content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -1,47 +1,66 @@
-===========================
-Manage multiple sales teams
-===========================
+====================
+Multiple sales teams
+====================
 
-In Odoo, you can manage several sales teams, departments or channels
-with specific sales processes. To do so, we use the concept of *Sales
-Channel*.
+In Odoo, you can manage several sales teams, departments, or channels, each with their own unique
+sales processes, using *Sales Teams*.
 
-Create a new sales channel
-==========================
 
-To create a new *Sales Channel*, go to :menuselection:`Configuration --> Sales Channels`.
+Create a new Sales Team
+=======================
 
-There you can set an email alias to it. Every message sent to that email
-address will create a lead/opportunity.
+To create a new *Sales Team*, go to :menuselection:`CRM --> Configuration --> Sales Teams` then
+click **Create**.
 
-.. image:: media/multi_sales_team01.png
-    :align: center
+On the creation page, you can set an *Email Alias* for your team. Every message sent to that email
+address will create a lead/opportunity in your pipeline. You can also choose whether you
+accept emails from *Everyone*, *Authenticated Partners*, or *Followers Only*.
 
-Add members to your sales channel
----------------------------------
+You can set up a *Domain* if you want to assign leads/opportunities to your team based on specific
+filters, such as country, language, etc. Set an *Invoicing Target* if your team has specific
+monthly revenue targets they need to achieve.
 
-You can add members to any channel; that way those members will see the
-pipeline structure of the sales channel when opening it. Any
-lead/opportunity assigned to them will link to the sales channel.
-Therefore, you can only be a member of one channel.
+.. image:: multiple_sales_teams/sales-team-creation.png
+   :align: center
+   :alt: Creating a Sales Team
 
-This will ease the process review of the team manager.
+Add members to your Sales Team
+------------------------------
 
-.. image:: media/multi_sales_team02.png
-    :align: center
+You can add team members to a Sales Team by clicking **Add** under the Assignment tab when editing
+the Sales Team's configuration page. Choose a salesperson by selecting them from the dropdown menu
+or creating a new salesperson. To ensure that your salespeople are not overworking, you can set a
+maximum number of leads that they can be assigned in a 30-day period.
 
-If you now filter on this specific channel in your pipeline, you will
-find all of its opportunities.
+.. image:: multiple_sales_teams/add-a-salesperson.png
+   :align: center
+   :alt: Adding a Salesperson
 
-.. image:: media/multi_sales_team03.png
-    :align: center
+You can add someone as a team member or a Team Leader to multiple Sales Teams, allowing them to
+easily check all of the pipelines they need access to.
```

Comment:
Removing "you" and improving wording

`One person can be added as a team member or Team Leader to multiple Sales Teams, allowing them to
access all of the pipelines that they need to.`